### ISSUE TITLE
Add py.typed to distribution

### ISSUE THREAD

**Issue by asford** [LuminosoInsight/ordered-set#77]
`ordered-set` is great, and even has type annotations. 👏 

Unfortunately, `mypy` is rather strict about [PEP 561](https://www.python.org/dev/peps/pep-0561/) and doesn't support use the type annotations in consuming applications. 😿 

It would be great to update the source distribution to:

```
ordered_set/
  __init__.py # current contents of ordered_set.py
  py.typed
```

So `mypy` and other type checkers will properly recognize the library.

**Comment by Eric-Arellano** [comment]
Hey @asford, check out https://github.com/LuminosoInsight/ordered-set/issues/72.

**Comment by asford** [comment]
🤯 How embarrassing! I totally failed to search. 

This would be closed by #75, and is a duplicate of #72
